src/main/java/thePackmaster/cards/warriorpack/Fall.java

```diff
@@ -0,0 +1,73 @@
+package thePackmaster.cards.warriorpack;
+
+import com.evacipated.cardcrawl.mod.stslib.damagemods.DamageModifierManager;
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.animations.VFXAction;
+import com.megacrit.cardcrawl.actions.common.GainBlockAction;
+import com.megacrit.cardcrawl.actions.utility.WaitAction;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.vfx.combat.WeightyImpactEffect;
+import thePackmaster.cardmodifiers.warriorpack.FrontDamage;
+import thePackmaster.cards.AbstractPackmasterCard;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+import static thePackmaster.util.Wiz.atb;
+
+public class Fall extends AbstractPackmasterCard {
+
+    public final static String ID = makeID(Fall.class.getSimpleName());
+
+    private static final int COST = 2;
+    private static final int BASE_DAMAGE = 6;
+
+    public Fall(){
+        super(ID, COST, CardType.ATTACK, CardRarity.UNCOMMON, CardTarget.ENEMY);
+        baseDamage = BASE_DAMAGE;
+        DamageModifierManager.addModifier(this, new FrontDamage());
+    }
+
+    @Override
+    public void use(AbstractPlayer p, AbstractMonster m) {
+        if (m != null) {
+            this.addToBot(new VFXAction(new WeightyImpactEffect(m.hb.cX, m.hb.cY)));
+            this.addToBot(new WaitAction(0.8F));
+        }
+        dmg(m, AbstractGameAction.AttackEffect.NONE);
+    }
+
+    public void applyPowers() {
+        this.baseDamage = AbstractDungeon.player.drawPile.size() + BASE_DAMAGE;
+        super.applyPowers();
+
+        if (!upgraded)
+            this.rawDescription = cardStrings.DESCRIPTION + cardStrings.EXTENDED_DESCRIPTION[0];
+        else
+            this.rawDescription = cardStrings.UPGRADE_DESCRIPTION + cardStrings.EXTENDED_DESCRIPTION[0];
+
+        this.initializeDescription();
+    }
+
+    public void calculateCardDamage(AbstractMonster mo) {
+        super.calculateCardDamage(mo);
```

Comment:
Doesn#t do the baseDamage calculation here. Not needed? I forget

Reply:
I don't understand ? I've tested with damage modifying powers (vuln/Vigor) it scales

Reply:
Is it still affected when hovering the monster with the card? If so, does it deal the correct damage?

Reply:
don't know why it did the form once but not twice

Reply:
Gotcha, all good then